client/src/main/scala/drt/client/components/FlightTableComponents.scala

```diff
@@ -50,20 +51,26 @@ object FlightTableComponents {
 
   def dateTimeComponent(): js.Function = (props: js.Dynamic) => {
     val dt = props.data.toString
-    if (dt != "") {
-      localDateTimeWithPopup(dt)
+    if (dt != "" && dt != "0") {
+      Try {
+        val sdate = SDate.parse(dt)
+        sdateLocalTimePopup(sdate)
+      }.recover {
+        case f =>
+        localDateTimeWithPopup(dt.toLong)
+      }
     } else {
       <.div.render
     }
   }
 
-  def localDateTimeWithPopup(dt: String): TagMod = {
-    if (dt.isEmpty) <.span() else localTimePopup(dt)
+  def localDateTimeWithPopup(dt: MillisSinceEpoch): TagMod = {
+    if (dt == 0) <.span() else localTimePopup(dt)
```

Comment:
should these not be Option[Long] so we don't have to make up bogus empty values everywhere?